runtime/include/chpl-comm-compiler-macros.h

```diff
@@ -53,7 +53,7 @@ void chpl_gen_comm_get(void *addr, c_nodeid_t node, void* raddr,
                        size_t size, int32_t commID, int ln, int32_t fn)
```

Comment:
Perhaps this could use `__attribute__((always_inline))` just before this declaration? Maybe put it in a macro & only use it for GCC and clang?

Our CHANGES.md says the runtime eliminated use of `__always_inline` in 1.12. Digging a bit, that happened in PR #1631  because `__always_inline` was always a workaround for problems building with GASNet, and we found a better solution.

The old way we defined it was

```
#ifdef __GNUC__
#define ___always_inline inline __attribute__((__always_inline__))
#else
#define ___always_inline inline
#endif
```